code/addons/a11y/package.json

```diff
@@ -56,7 +56,8 @@
     "dist/**/*",
     "README.md",
     "*.js",
-    "*.d.ts"
+    "*.d.ts",
+    "!src/**/*"
```

Comment:
Just curious, how was `src` being included in the first place?  Was it the `"*.js"`?

Reply:
Yes, the *.js match would include things in src such as `src/__mockdata__/someStory.js` and things like that.
The biggest chunk that got shaved off doesn't come from that, it comes from changes like this:

```diff
-    "template/**/*",
+    "template/cli/**/*",
```

because we were introducing **a lot** of example stories that only matter for our sandboxes (a little over 800kb)